packages/ui/resolution/ResolutionDecorator.js

```diff
@@ -28,6 +28,29 @@ const defaultConfig = {
 	 */
 	dynamic: true,
 
+	/**
+	 *
+	 * Determines how to get the best match screen type of current resolution. When set to `true`, the screen type is the one closest to the screen resolution without going over.
+	 * When set to `false`, the screen type is the one closest to the screen resolution without going below.
+	 *
+	 * @type {Boolean}
+	 * @private
+	 * @memberof ui/resolution.ResolutionDecorator.defaultConfig
+	 */
+	findScreenTypeWithoutGoingOver: false,
+
+	/**
+	 *
+	 * Determines how to calculate font-size. When set to `scale` and the screen is in `landscape` orientation, calculates font-size linearly based on screenTypes.
+	 * When set to `normal`, the font-size will be the pxPerRem value of the best match screen type.
+	 *
+	 * @type {String}
```

Comment:
For this case, we should use
```suggestion
	 * @type {('normal'|'scale')}
```